ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -429,13 +429,8 @@ extension ImagePickerSheetController: UICollectionViewDelegate {
         if let maximumSelection = maximumSelection {
             if selectedAssetIndices.count >= maximumSelection,
                 let previousItemIndex = selectedAssetIndices.first {
-                    let deselectedAsset = selectedAssets[previousItemIndex]
-                    delegate?.controller?(self, willDeselectAsset: deselectedAsset)
```

Comment:
The crash occurs when you set `maximumSelection` to 1 and then try to select a different image after selecting one already (all before dismissing the image picker sheet ).